src/core/Akka.Tests.Shared.Internals/AkkaSpec.cs

```diff
@@ -39,13 +40,13 @@ public abstract class AkkaSpec : Xunit2.TestKit    //AkkaSpec is not part of Tes
 
         private static int _systemNumber = 0;
 
-        public AkkaSpec(string config)
-            : this(ConfigurationFactory.ParseString(config).WithFallback(_akkaSpecConfig))
+        public AkkaSpec(string config, ITestOutputHelper output = null)
```

Comment:
`ITestOutputHelper` can be added only by the test runner and only as a constructor param for test class. It's out of our control.